CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_lever v0.7.0
```

Comment:
Request to add some of the comments from the source PR to this one as well so users are aware of those upstream changes as well.